internal/pkg/cli/prompt/interactive/interactive.go

```diff
@@ -54,9 +54,7 @@ func (p *Prompt) IsInteractive() bool {
 }
 
 func (p *Prompt) Printf(format string, a ...interface{}) {
-	if _, err := fmt.Fprintf(p.stdout, format, a...); err != nil {
-		panic(err)
-	}
+	_, _ = fmt.Fprintf(p.stdout, format, a...)
```

Comment:
Ak je v teste chyba, tak zatvarame stdin/stdout a moze nastat tento panic - a potom sa neukaze hlavna chyba.
Toto je len nejaky logovaci vystup v interaktivnom dialogu, chybu mozeme ignorovat.

Reply:
Myslíš, že by to dávalo smysl hodit do komentáře?

Reply:
ok :+1:

Reply:
https://github.com/keboola/keboola-as-code/pull/823/commits/487f20bb6cb6c8b6517b96371e5e05f6b4bcd108